jhrbva/Caravan
# GET request endpoint to fetch rest stops

## Description
implemented a way to show the user rest stops added if any from the db

## Related Issues
Link any related issues, like this:
https://caravanning.atlassian.net/browse/CA-93

## Testing
1. Run the server ``` node server.js ```
2. Open or Download [Postman](https://www.postman.com/downloads/).
3. To test the functionality create a ``` GET ``` endpoint ``` localhost:8080/reststop/1 ```
4. Once the above steps are completed, you may click ``` Send ``` (the blue button located towards the top of the application). 
![Screen Shot 2020-04-02 at 7 00 28 PM](https://user-images.githubusercontent.com/12536700/78307796-c3067f00-7514-11ea-8096-e55d2d329529.png)

5. This PR is working correctly if you see the following response:
![Screen Shot 2020-04-02 at 7 00 43 PM](https://user-images.githubusercontent.com/12536700/78307810-cf8ad780-7514-11ea-867e-43bf31c601e8.png)

## How are you feeling?
😀
